src/d/actor/d_a_ship.cpp

```diff
@@ -5,460 +5,4718 @@
+                procStartModeWarp_init();
+            }
+            else if (this->m034D == 16) {
+                procStartModeThrow_init();
+            }
+            else {
+                procWait_init();
+            }
+        }
+
+        if ((dStage_stagInfo_GetSTType(dComIfGp_getStageStagInfo()) == 7) || !strcmp(dComIfGp_getStartStageName(), "Hyrule") || !strcmp(dComIfGp_getStartStageName(), "Ocean")) {
+            this->m03CC = 4;
+        }
+        else {
+            this->m03CC = 3;
+        }
+        
+        this->mAcch.Set(fopAcM_GetPosition_p(this), fopAcM_GetOldPosition_p(this), this, this->m03CC, &this->mAcchCir[0], fopAcM_GetSpeed_p(this), fopAcM_GetAngle_p(this), fopAcM_GetShapeAngle_p(this));
+        
+        this->mAcchCir[0].SetWall(0.0f, 250.0f);
+        this->mAcchCir[1].SetWall(75.0f, 250.0f);
+        this->mAcchCir[2].SetWall(150.0f, 250.0f);
+        this->mAcchCir[3].SetWall(-600.0f-this->current.pos.y, 250.0f);
+
+        this->mAcch.ClrWaterNone();
+        this->mAcch.SetWaterCheckOffset(10000.0f);
+
+        this->attention_info.flags = 0;
+        this->attention_info.position = this->current.pos;
+
+        this->gravity = -2.5f;
+        this->maxFallSpeed = -150.0f;
+        this->mGridID = fopAcM_create(PROC_GRID, 1, &this->current.pos, -1, &this->current.angle);
+
+        if (this->mGridID == fpcM_ERROR_PROCESS_ID_e) {
+            return cPhs_ERROR_e;
+        }
+
+        this->mTactWarpPosNum = -1;
+        this->m03E0 = 10000.0f;
+        this->mStts.Init(0xF0, 0, this);
+
+        dCcD_Cyl* cyl = mCyl;
+        for (int i = 0; i < 3; i++, cyl++) {
+            cyl->Set(cyl_src);
+            cyl->SetStts((&this->mStts));
+        }
+
+        this->mCyl[1].SetR(95.0f);
+        this->mSph.Set(sph_src);
+        this->mSph.SetStts(&this->mStts);
+
+        this->cull.box.min.x = -325.0f;
+        this->cull.box.min.y = -50.0f;
+        this->cull.box.min.z = -325.0f;
+        this->cull.box.max.x = 325.0f;
+        this->cull.box.max.y = 570.0f;
+        this->cull.box.max.z = 240.0f;
+        
+        fopKyM_create(PROC_WIND_ARROW, (int)this, 0, 0, 0);
+
+        offStateFlg(daSFLG_UNK2_e);
+        this->mAcch.CrrPos(*dComIfG_Bgsp());
+
+        setRoomInfo();
+
+        this->m03F4 = getWaterY();
+        
+        if (this->m034C == 13) {
+            this->current.pos.y = this->m03F4 + 5000.0f;
+        }
+        else if (this->m034C == 16) {
+            this->current.pos.y = this->m03F4 + 2500.0f;
+        }
+
+        mDoMtx_stack_c::transS(this->current.pos.x, this->current.pos.y, this->current.pos.z);
+        mDoMtx_ZXYrotM(mDoMtx_stack_c::get(), this->shape_angle.x, this->shape_angle.y, this->shape_angle.z);
+        
+        this->m0298->getModel()->setBaseTRMtx(mDoMtx_stack_c::get());
+        this->m0298->play(NULL, 0, 0);
+        this->m0298->calc();
+        
+        PSMTXMultVec(this->m0298->getModel()->getAnmMtx(10), &l_tiller_top_offset, &this->m0438);
```

Comment:
```suggestion
        cMtx_multVec(this->m0298->getModel()->getAnmMtx(10), &l_tiller_top_offset, &this->m0438);
```
If you see a `PSMTX` function being called, it's usually an inline in m_Do_mtx.h